html/factsheet-actor.ejs

```diff
@@ -66,20 +66,28 @@
             </div>
           </div>
 
-          <div class="c-area-stacked-chart">
-            <span class="title">TOP Source Municipalities</span>
-            <div class="js-municipalities-top">
+          <div class="container-top-tables">
+
+            <div class="c-area-stacked-chart">
+              <span class="title -small">TOP Source Municipalities in 2015</span>
+              <div class="c-area-table -top-table">
+                <div class="js-municipalities-top">
+                </div>
+              </div>
             </div>
-          </div>
 
-          <div class="c-area-stacked-chart -second ">
-            <span class="title">TOP destination COUNTRIES</span>
-            <div class="js-destination-top">
+            <div class="c-area-stacked-chart -second ">
```

Comment:
`-second` are you using this class for something? If you need it to separate from other area stack, use brother selector.